homeassistant/components/light/tplink.py

```diff
@@ -195,5 +197,9 @@ def get_features(self):
             self._supported_features += SUPPORT_BRIGHTNESS
         if self.smartbulb.is_variable_color_temp:
             self._supported_features += SUPPORT_COLOR_TEMP
+            self._min_mireds = kelvin_to_mired(
+                self.smartbulb.valid_temperature_range[0])
```

Comment:
You've swapped 0 and 1. It used to be min mireds was `range[1]`

Reply:
Never mind, read your PR description

Reply:
Indeed, I thought that min_mireds should be based on min kelvins and so, but apparently that is a wrong assumption and the original was correct. I'll change that.